src/main/Validator.ts

```diff
@@ -88,6 +89,7 @@ const configDataSchemaV2 = Joi.object<ConfigV2>({
     useSpellChecker: Joi.boolean().default(true),
     enableHardwareAcceleration: Joi.boolean().default(true),
     autostart: Joi.boolean().default(true),
+    hideOnStart: Joi.boolean().default(false),
```

Comment:
same thing with V2, you only need on V3 and after

Reply:
Fixed to.